tests/theme.rs

```diff
@@ -1,8 +1,3 @@
-
-
-
-
-
```

Comment:
I added these empty lines because in some scenarios the first five lines of a file are extracted for the error (if I remember correctly, I assumed, when color-eyre couldn't find the proper lines, for some reason). In this case, the structure of the error could change when the content of the first five lines changes (at least I was assuming this / I couldn't rule this out completely), which would make the saved test data incompatible.

As far as I remember, this should not be very likely to happen (because we only check ANSI escape sequences), however. Unfortunately, I don't remember the details anymore, so I'm not 100% sure.

I guess rustfmt removed these empty lines. Empty comment lines would be as good as empty lines.

Reply:
I ended up running into too many issues trying to handle testing the output of the source context with `RUST_BACKTRACE=full`, for example, on my machine I'd get the source for `FnOnce`'s frame because I had the `rust-src` component installed but this wasn't on by default in CI so it would not be able to find or display the source code. I think I was able to fix that one but there were other places it still broke that just convinced me to switch to `RUST_BACKTRACE=1` and call it a day. Hopefully that means that now the 5 lines not showing up wont matter?

Reply:
Ah, I missed that you switched to `RUST_BACKTRACE=1`. Yes, this shouldn't matter much / be okay. The current line is differently styled than the surrounding source lines, but the surrounding lines just were not styled at all (as far as I remember).